### PR TITLE
Github Actions user doc.

### PR DESCRIPTION
**Pull request by nsabri1**
https://trello.com/c/1k2eVCZS/670-good-enough-user-docs

### CODE REVIEW

File: source/documentation/index.md [hunk 1]
```diff
@@ -163,6 +163,21 @@ $ kubectl -n <namespace> port-forward <pod> 5000:5001
 
 ### Github Actions (CI)
 
+With GitHub Actions, we can now automatically build and publish our docker image to our private GOV.UK production AWS Elastic Container Repository.
+
+The GitHub Actions workflow CI yaml file can be found within each repository in the following location: `.github/workflows/ci.yaml`
+
+When a user merges a branch to main, the GitHub Action Workflow will be triggered, the defined job within the ci.yaml file will then start.
```
Comment (jonathanglassman) [inline source/documentation/index.md:170]: GitHub Action => GitHub Actions

File: source/documentation/index.md [hunk 2]
```diff
@@ -163,6 +163,21 @@ $ kubectl -n <namespace> port-forward <pod> 5000:5001
...
+Within the job there is steps which will checkout the code, build a new Docker and then push the new Dcoker image to GOV.UK production AWS Elastic Container Repository.
```
Comment (jonathanglassman) [inline source/documentation/index.md:171]: - there is steps => there are steps
- build a new Docker => build a new Docker image
- then push the new Dcoker image => then push that new Docker image

File: source/documentation/index.md [hunk 3]
```diff
@@ -163,6 +163,21 @@ $ kubectl -n <namespace> port-forward <pod> 5000:5001
...
+Each new image will be tagged with merge commit GitHub simple hashing algorithm(SHA).
```
Comment (jonathanglassman) [inline source/documentation/index.md:172]: add space between "simple hashing algorithm" and "(SHA)"

File: source/documentation/index.md (hunk 1 above)
Comment (nsabri1) [reply source/documentation/index.md:170]: Good spot, Thanks.

File: source/documentation/index.md [hunk 4]
```diff
@@ -163,6 +163,21 @@ $ kubectl -n <namespace> port-forward <pod> 5000:5001
...
+
+You can view the status of GitHub Action CI on `https://github.com/alphagov/<application repo>/actions`, workflow is named `Build and publish to ECR`.
```
Comment (jonathanglassman) [inline source/documentation/index.md:174]: - GitHub Action CI => GitHub Actions CI 
- recommend splitting sentence i.e. "You can view the status of GitHub Actions CI on `https://github.com/alphagov/<application repo>/actions`. The workflow is named `Build and publish to ECR`.

File: source/documentation/index.md [hunk 5]
```diff
@@ -163,6 +163,21 @@ $ kubectl -n <namespace> port-forward <pod> 5000:5001
...
+
+#### Sensitive data
+AWS credentials have been stored as Github Organisation level secrets and have been made availble to individual repos, if a new repo requires acccess to secrets please contact Replatforming team.
```
Comment (jonathanglassman) [inline source/documentation/index.md:177]: Recommend splitting sentence, correcting a couple of typos and saying how to contact the replatforming team with a link. For example:

"AWS credentials are stored as Github Organisation level secrets and have been made available to individual repos. If a new repo requires access to secrets, please [contact the Replatforming team on Slack](https://gds.slack.com/archives/C013F737737).

**Review by jonathanglassman — CHANGES_REQUESTED** [review]
Looks good, just a few comments

File: source/documentation/index.md (hunk 2 above)
Comment (fredericfran-gds) [reply source/documentation/index.md:171]: Thanks, will do.

File: source/documentation/index.md (hunk 3 above)
Comment (fredericfran-gds) [reply source/documentation/index.md:172]: Thanks, will do.

File: source/documentation/index.md (hunk 4 above)
Comment (fredericfran-gds) [reply source/documentation/index.md:174]: thanks

File: source/documentation/index.md (hunk 5 above)
Comment (fredericfran-gds) [reply source/documentation/index.md:177]: Thanks.

Comment (nsabri1) [reply source/documentation/index.md]: Done.

**Review by jonathanglassman — APPROVED** [review]

**Pull request merged**